benchexec/tools/cpachecker.py

```diff
@@ -182,6 +182,30 @@ def isOutOfNativeMemory(line):
             status = result.RESULT_ERROR
         return status
 
+    def determine_result_for_property(self, returncode, returnsignal, output, isTimeout, requiredProperty):
+        """
+        @param returncode: code returned by CPAchecker
+        @param returnsignal: signal, which terminated CPAchecker
+        @param output: the output of CPAchecker
+        @param requiredProperty: property, for which status is expected
+        @return: status of CPAchecker after executing a run for specified property
+        """
+        for line in output:
+            # 	Property __VERIFIER_error_linux_kernel_rcu_srcu: UNKNOWN
+            match = re.match('.*Property (.*): (TRUE|FALSE|UNKNOWN).*', line)
+            if match:
+                foundProperty = match.group(1)
+                verdict = match.group(2)
+                if foundProperty in requiredProperty:
```

Comment:
It would be better to match the property name exactly. Why is this not possible?